### DISCUSSION TITLE
Permanence on the details toggle

### DISCUSSION THREAD

**Discussion by vaughn-taylor** [craftcms/cms#14991] [Ideas] [answered]
Can a cookie be set for the sidebar toggle on an entry page so that we don't need to click it into either open or closed every time? The meta data is useful for sure, but I don't feel that it needs to be present by default.

**Comment by ryanleichty** [comment]
Yes! I was just going to create a feature request for this too. When using the card view for matrix fields (or when opening any kind of entry pane), I don't want to have to collapse the sidebar every time I open another entry.

![CleanShot 2024-05-22 at 14 33 26@2x](https://github.com/craftcms/cms/assets/17421429/56a41b9b-0bf9-4eb5-bb3a-d2e24754eea1)

**Comment by TheFunkyMonk** [comment, marked as answer]
I requested this here too, hopefully this gets some attention. https://github.com/craftcms/cms/discussions/14798
